### DISCUSSION TITLE
Support for  jaeger remote sampler

### DISCUSSION THREAD

**Discussion by sambaatteamf1** [open-telemetry/opentelemetry-java-instrumentation#1950] [Q&A] [answered]
It looks like the tracer config does not allow setting a specific sampler. The sampler is chosen based on the value of otel.config.sampler.probability(See https://github.com/open-telemetry/opentelemetry-java/blob/7c0b45209761443c2dd7f9a659ba6e99aded222b/sdk/trace/src/main/java/io/opentelemetry/sdk/trace/config/TraceConfigBuilder.java#L132). 

I want to configure the javaagent to use jaeger remote sampler (https://github.com/open-telemetry/opentelemetry-java/tree/7c0b45209761443c2dd7f9a659ba6e99aded222b/sdk-extensions/jaeger-remote-sampler), so that i can filter traces for health check.

Are there any plans to support such a configuration ?

**Comment by trask** [comment, marked as answer]
hi @sambaatteamf1! I don't think there's any open issue for this, if you want to open one that would be great, it seems like a good feature for the javaagent to support

**Reply by sambaatteamf1** [reply]
Sure. https://github.com/open-telemetry/opentelemetry-java-instrumentation/issues/1953
